app/javascript/components/App.js

```diff
@@ -13,19 +12,42 @@ import Dash from "./components/Dash";
 import LandingPage from "./components/LandingPage";
 import Header from "./components/Header";
 import YourTrips from "./pages/YourTrips";
+import CreateSight from "./pages/CreateSight";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      trips: [],
+      trips: false,
+      businesses: false,
```

Comment:
Generally for defaulting values, they should be same data type as the data it is intended to store or `null` or `undefined`. For something like trips where I expect to see a collection of things, consider empty array or null.

Reply:
ah okay thank you

Reply:
will change that